process/transaction/shardProcess.go

```diff
@@ -789,6 +783,26 @@ func (txProc *txProcessor) processInnerTx(
 
 	txFee := txProc.computeInnerTxFee(innerTx)
 
+	innerTxHash := originalTxHash
```

Comment:
can you make a separate function for this ? 

If would be easier to understand - otherwise you look at this and you do not seem that it is ok.

Also, can you rename originalTxHash - to relayerTxHash ?

Or it is relayerTxHash always ? If it is not, than it can stay originalTxHash.

Reply:
added new method + kept originalTxHash.. in case of relayedV3, it can be either relayerTxHash either innerTxHash